clang-tools-extra/clang-tidy/bugprone/UnsafeFunctionsCheck.cpp

```diff
@@ -7,6 +7,8 @@
 //===----------------------------------------------------------------------===//
 
 #include "UnsafeFunctionsCheck.h"
+#include "../utils/Matchers.h"
```

Comment:
(This include is superfluous, ClangTidyCheck automatically pulls the related header in. Yes, I'm always burnt by clangd with this as well. :wink:)

Reply:
Not in this case! utils/Matchers.h is not included in ClangTidyCheck (but it is included in UnsafeFunctionsCheck.h, so... whoops)